app/controllers/concerns/accountable.rb

```diff
@@ -14,8 +14,23 @@ def create_user_account
     log e
   end
 
+  def update_account_login_stats
+    return unless login_stats.present? && login_type.in?(AccountLoginStat::LOGIN_TYPES)
+
+    login_stats.update("#{login_type}_at" => Time.zone.now)
+  end
+
   private
 
+  def login_stats
+    @login_stats ||= AccountLoginStat.find_or_initialize_by(account_id: @current_user.account.id) \
+      if @current_user.account.present?
```

Comment:
I believe RuboCop might have had too much motor oil. 🍸 
```suggestion
    @login_stats ||=
      @current_user.account.present? &&
      AccountLoginStat.find_or_initialize_by(account_id: @current_user.account.id)
```
Give this a shot. I think there's something weird with line length here, RuboCop should not be recommending inline if modifier if it makes the line too long.